simple_salesforce/bulk.py

```diff
@@ -154,11 +156,16 @@ def _get_batch_results(self, job_id, batch_id, operation):
                                   headers=self.headers)
 
         if operation == 'query':
-            url_query_results = "{}{}{}".format(url, '/', result.json()[0])
-            query_result = call_salesforce(url=url_query_results, method='GET',
-                                            session=self.session,
-                                            headers=self.headers)
-            return query_result.json()
+            result_json = []
+            for batch_result in result.json():
+                url_query_results = "{}{}{}".format(url, '/', batch_result)
+                query_result = call_salesforce(url=url_query_results,
+                                                method='GET',
+                                                session=self.session,
+                                                headers=self.headers)
+                result_json += query_result.json()
+
+            return result_json
```

Comment:
Is this the same return value as before? It seems like this is now a `list` whereas before it was the JSON.